### PR TITLE
LPS-173273 | Test Fix | Master

### PR DESCRIPTION
**Pull request by JerryAdrianoo**
https://issues.liferay.com/browse/LPS-173273

### CODE REVIEW

**Comment by JerryAdrianoo** [comment]
ci:test:sf

**Comment by evanilson-santana** [comment]
start review

**Comment by evanilson-santana** [comment]
Hey @JerryAdrianoo great job on this test fix. The next time, can you please attach the index file to the ticket?  Please see my comments below and make the changes. If you have any questions feel free to let me know.

File: modules/apps/object/object-test/src/testFunctional/paths/ObjectRelationships.path [hunk 1]
```diff
@@ -20,6 +20,11 @@
 		<td>//button[@class = 'dropdown-item']//div[contains(@class, 'd-flex')][contains(text(),'${key_parameterOption}')]</td>
 		<td></td>
 	</tr>
+	<tr>
+		<td>SELECT_PARAMETER_OPTION_IN_RELATIONSHIP</td>
+		<td>//div[@id='clay-dropdown-menu-3']//button[@class = 'dropdown-item']//div[contains(text(),'${key_parameterOption}')]</td>
+		<td></td>
+	</tr>
```
Comment (evanilson-santana) [inline modules/apps/object/object-test/src/testFunctional/paths/ObjectRelationships.path:27]: This path is not necessary for this step. You can use the same path on the macro. See: `"ObjectRelationships#SELECT_PARAMETER_OPTION"`. Try it and see if works.

File: modules/apps/object/object-test/src/testFunctional/macros/ObjectAdmin.macro [hunk 2]
```diff
@@ -289,8 +289,12 @@ definition {
 				value1 = "${name}");
 		}
 
-		if (isSet(parameterOption)) {
-			ObjectRelationships.editRelationshipsParameter(parameterOption = "${parameterOption}");
+		if (isSet(parameter)) {
```
Comment (evanilson-santana) [inline modules/apps/object/object-test/src/testFunctional/macros/ObjectAdmin.macro:292]: When you changed this variable you forgot to pass it on the test case. Can you change this: `parameterOption = "Self Relationship",` `for this: parameter = "Self Relationship",`

File: modules/apps/object/object-test/src/testFunctional/tests/ObjectRelationships.testcase [hunk 3]
```diff
@@ -867,7 +867,11 @@ definition {
 		task ("Verify it's possible to edit a relationship parameter") {
 			ObjectAdmin.goToRelationshipsDetails(fieldName = "Relationship with Custom Object");
 
-			ObjectRelationships.editRelationshipsParameter(parameterOption = "Relationship with Custom Object");
+			Click(locator1 = "ObjectRelationships#SELECT_PARAMETER_FORM");
+
+			ClickNoError.clickNoMouseOver(
+				key_parameterOption = "Relationship with Custom Object",
+				locator1 = "ObjectRelationships#SELECT_PARAMETER_OPTION_IN_RELATIONSHIP");
```
Comment (evanilson-santana) [inline modules/apps/object/object-test/src/testFunctional/tests/ObjectRelationships.testcase:874]: And here you must pass this path "ObjectRelationships#SELECT_PARAMETER_OPTION"

**Comment by timpak** [comment]
@JerryAdrianoo is there additional work needed to finish this PR?

**Comment by JerryAdrianoo** [comment]
> @JerryAdrianoo is there additional work needed to finish this PR?

Hi, Tim. 
This PR is no longer needed. This error has now been fixed and this fix is ​​obsolete.

**Pull request closed without merging**